test/templates/api/projects/instance-upgrade/expected-state.json

```diff
@@ -19,7 +19,8 @@
             "parameters": {
               "param1": "modified",
               "param2": "value2",
-              "param3": "value3"
+              "param3": "value3",
+              "id": "123456"
             }
```

Comment:
A po teste mame novy stav konfiguracie, ale ID sa tam automaticky doplnilo.

V sablonach neriesime poradie fields, pridalo sa to nakoniec.
- Totiz ani Jsonnet nezachovava poradie fields, zoradi ich abecedne.
- To ^^^ by sa v Jsonnet libke dalo upravit, ale odlozili sme to na neskor, a zatial sa nikto nestazoval.